packages/headless/src/api/search/search/result.ts

```diff
@@ -172,4 +172,9 @@ export interface Result {
    * TopResult
    */
   rankingModifier?: string;
+
+  /**
+   * The associated `searchUid` of the response where the result was fetched.
+   */
+  searchUid: string;
```

Comment:
Okay, I think I'll go with this approach

Reply:
>  it could be convenient to just concatenate the searchUid and uniqueId

I think searchUid directly is fine, since there could be other instances where you want to link a result object with a given request, we never know!

Reply:
Could this property instead be "moved" to the `InteractiveResult` controller ?

So, instead of tackling this on the raw object, we encourage users to use that controller a bit more ?

Because IMO, this unique identifier problem is borderline a UI problem only, and not 100% headless related.

Reply:
Hum interactive result is something else, for a component like resultLink, it's the result list which need the property

Reply:
I'm not sure if we should've been exposing the API object in the first place. Perhaps in a later version, we could have `ApiResult` and `Result` separately?

Reply:
I personally feel like it's nicer to have the searchUid directly on the result.